ci/jenkinsfiles/build.groovy

```diff
@@ -260,6 +247,52 @@ pipeline {
         }
       }
     }
+    stage('Run Functional Tests') {
+      steps {
+        container('maven') {
+          script {
+            gitHubBuildStatus.set('ftests')
+            try {
+              retry(3) {
+                nxNapps.runFunctionalTests(
+                  "${FRONTEND_FOLDER}", "--nuxeoUrl=http://preview.${PREVIEW_NAMESPACE}.svc.cluster.local/nuxeo"
+                )
+              }
+            } catch(err) {
+              throw err
+            } finally {
+              //retrieve preview logs
+              nxKube.helmGetPreviewLogs("${PREVIEW_NAMESPACE}")
+              cucumber (
+                fileIncludePattern: '**/*.json',
+                jsonReportDirectory: "${FRONTEND_FOLDER}/target/cucumber-reports/",
```

Comment:
Hello @PGoncalvesNX 
the report wasn't generated
![Screenshot 2021-04-09 at 13 50 50](https://user-images.githubusercontent.com/59877775/114175839-bc39f580-993a-11eb-9f76-89437412f1d7.png)

Reply:
What do I need to change in the code?

Reply:
> What do I need to change in the code?

perhaps , we should update the content of the package.json file
but I'm not sure 🤔 ( do you have the report when running the test locally ?)

Reply:
Yes, I do 😬